src/lib/openapi/spec/client-features-schema.ts

```diff
@@ -13,23 +13,34 @@ export const clientFeaturesSchema = {
     $id: '#/components/schemas/clientFeaturesSchema',
     type: 'object',
     required: ['version', 'features'],
+    description:
+        'Configuration data for server-side SDKs for evaluating feature flags.',
     properties: {
         version: {
             type: 'number',
+            description:
+                'A version number for the format used in the response. Most Unleash instances now return version 2, which includes segments as a separate array',
```

Comment:
Discussion: should this property be an enum instead of a number? Maybe, maybe not. Do you have any thoughts? If all we have is 1 and 2, then it's not very helpful to put 564, for instance. Not very important in a response schema, but interesting none the less.